waiter/src/waiter/util/utils.clj

```diff
@@ -133,18 +133,13 @@
 (defn stringify-elements
   [k v]
   (cond
-    (vector? v) (mapv (partial stringify-elements k) v)
-    (keyword? v) (keyword->str v)
+    (contains? #{true false nil} v) v
+    (map? v) v
+    (number? v) v
+    (sequential? v) (mapv (partial stringify-elements k) v)
     (instance? DateTime v) (du/date-to-str v)
-    (instance? UUID v) (str v)
-    (instance? Pattern v) (str v)
-    (instance? PersistentQueue v) (vec v)
-    (instance? ManyToManyChannel v) (str v)
-    (instance? Process v) (str v)
-    (instance? ValidationError v) (str v)
-    (= k :time) (str v)
-    (symbol? v) (str/join "/" ((juxt namespace name) v))
-    :else v))
+    (keyword? v) (keyword->str v)
+    :else (str v)))
```

Comment:
We just call `.toString` on anything else.

Note that symbols don't need special handling. The old code is actually broken in this respect.

`(str 'x/y)` → `"x/y"`
`(str 'z)` → `"z"`
`(str/join "/" ((juxt namespace name) 'z))` → `"/z"`

I can't see any reason why we'd want to prepend `/` in the last case. That just seems wrong.